Evaluation/Input/Content/References.md

```diff
@@ -11,9 +11,9 @@ Aug;60(2):183-90
 
 **DrugBank DB00176** (https://www.drugbank.ca/drugs/DB00176)
 
-**FDA_ClinPharmReview LuvoxCR** FDA\_ClinPharmReview LuvoxCR, NDA 22-033, FDADrug\_42.pdf, website: digitalcollections.ohsu.edu/downloads/6m311p63g?locale=en
+**FDA_ClinPharmReview LuvoxCR** FDA\_ClinPharmReview LuvoxCR, NDA 22-033, FDADrug\_42.pdf, website: http://digitalcollections.ohsu.edu/downloads/6m311p63g?locale=en
```

Comment:
@pavelbal without http:// or https:// it is not recognized as link by markdown and is not clickable